main.less

```diff
@@ -1,32 +1,25 @@
+/*Font family for errorPanel*/
+@fontFamily: Helvetica, Arial, "Meiryo UI", "ＭＳ Ｐゴシック", "MS PGothic", sans-serif;
```

Comment:
I kept the fonts consistent with [this](https://github.com/humphd/brackets/blob/bramble/src/styles/brackets_fonts.less#L83). Do you think I should remove it?